Combat Rules/1_Combat_Rules.txt

```diff
@@ -0,0 +1,392 @@
+GM's patience and willingness to believe your character can do everything you 
+want in a negligible timescale, there is no limit to the number of Free Actions 
+a character may take, and they may be used at any time.
+
+===== Reactions =====
+
+	Certain abilities or circumstances will allow you do something 
+in response to a triggering event. Doing so requires using a special kind 
+of Action called a Reaction. Reactions occur at the immediate moment 
+that they are triggered, regardless of whose turn it is. Common uses of 
+Reactions include Opportunity Attacks (described in "Melee Combat", above) 
+and Held Actions, below. After the Reaction occurs, play continues as normal. 
+If it was an opponent's turn, they continue their turn. 
+
+	A character may use up to three Reaction per round, regardless of how 
+many viable trigger circumstances occur.
+
+===== Held Actions =====
+
+	A type of Reaction. A player may choose to forgo using up to 2 Actions at 
+the end of their turn. The player must declare to their DM what those actions 
+are to be triggered by, and then play continues as normal. If the trigger occurs 
+before the player's next turn, the player gets to immediately use their 1 or 2 
+actions. If the trigger does not occur before the player's turn, they lose their
+held action(s) and start their turn as normal. 
+
+	For example, let's say my Character and his/her friends are walking along a 
+ledge. I know that one of the Characters in the party is clumsy so I have my 
+Character move along the ledge, and I ask the DM if I can save my last action to 
+try and catch the clumsy Character, should they trip. The DM approves my plan 
+and play continues. On the clumsy Character's turn, they start to scoot across
+the ledge and trip! As soon as they trip, my held action triggers and my 
+Character saves them from falling off the ledge and their turn continues. If 
+they had not tripped, my held action would've been lost at the start of my 
+following turn.
+
+===== Example Actions =====
+
+	Below is a quick list of examples of things a character might want to do and 
+how many Actions that would take out of their turn. Your character isn't limited 
+to only these options; you can try to do almost anything you can think of, as 
+long as what you're trying to do isn't an ability that needs to be unlocked 
+elsewhere. If you'd like to do something that isn't on this list, your Game 
+Master will decide how many Actions it will take.
+
+Shooting: 1 Action
+Aiming: 1 Action
+Drawing a weapon: 1 Action
+Reloading: listed in weapon stats
+Suppressing an Area/Target: 1 Action
+Changing fire mode: Free Action
+
+Moving your Move Speed: 2 Actions
+Hunkering Down in cover: 1 Action
+Laying down: 1 Action
+Standing up from laying down: 1 Action
+Crouching: Free Action
+Standing up from crouching: 1 Action
+
+Talking: Free Action
+Performing a Skill Check (other than Getting Bearings): 1 Action
+Spotting(PER) checks to look for enemies: Free Action
```

Comment:
I was under the impression Awareness to notice enemies was free, but Spotting to seek out enemies, such as from underbrush, was an Action.

Reply:
I think you're right; I'll change it.